lib/HLSL/DxilGenerationPass.cpp

```diff
@@ -2908,7 +2908,7 @@ class DxilEmitMetadata : public ModulePass {
       unsigned ValMajor = 0;
       unsigned ValMinor = 0;
       M.GetDxilModule().GetValidatorVersion(ValMajor, ValMinor);
-      if (ValMajor == 1 && ValMinor == 0) {
+      if (ValMajor == 1 && ValMinor <= 1) {
         patchValidation_1_0(M);
```

Comment:
should probably refactor this; we want to remove extra metadata in all cases; it happens to be that patching for 1.0 validation also requires that